docs/setup/setup.md

```diff
@@ -26,9 +26,16 @@
   ```yaml
```

Comment:
Enter the **master address** or path to kubeconfig file in controller.yaml

Reply:
ok